components/activityInput/ActivityInputDialog.js

```diff
@@ -238,6 +238,8 @@ function ActivityInputDialog({ fullScreen, show, onClose }) {
       const stats = {
         activityLogEntriesCount: firebase.firestore.FieldValue.increment(1),
         activityLogEntriesLatestTimestamp: firebase.firestore.FieldValue.serverTimestamp(),
+        weeklyActivitiesCount: firebase.firestore.FieldValue.increment(1),
```

Comment:
Are the `weeklySomethingCount` what Gideon had as `current` in his POC?

Reply:
yes, serve the same purpose.

Reply:
We might want to keep the naming the same just for clarity and to make comparisons easier (ex: checking the `actionPlan.current.goals` with `actionPlan.target.goals`. Let me know your thoughts